README.md

```diff
@@ -1,17 +1,40 @@
-![SST](http://sst-simulator.org/img/sst-logo-small.png)
+![SST](http:g/sst-simulator.org/img/sst-logo-small.png)
 
 # Structural Simulation Toolkit (SST)
 
-#### Copyright (c) 2009-2021, National Technology and Engineering Solutions of Sandia, LLC (NTESS)
+### Copyright (c) 2009-2021, National Technology and Engineering Solutions of Sandia, LLC (NTESS)
 
 ---
 
 The Structural Simulation Toolkit (SST) was developed to explore innovations in highly concurrent systems where the ISA, microarchitecture, and memory interact with the programming model and communications system. The package provides two novel capabilities. The first is a fully modular design that enables extensive exploration of an individual system parameter without the need for intrusive changes to the simulator. The second is a parallel simulation environment based on MPI. This provides a high level of performance and the ability to look at large systems. The framework has been successfully used to model concepts ranging from processing in memory to conventional processors connected by conventional network interfaces and running MPI.
 
 ---
 
-Visit [sst-simulator.org](http://sst-simulator.org) to learn more about SST.
+Visit [sst-simulator.org](http:g/sst-simulator.org) to learn more about SST.
 
-See [Contributing](https://github.com/sstsimulator/sst-core/blob/devel/CONTRIBUTING.md) to learn how to contribute to SST.
+See [Contributing](https:g/github.com/sstsimulator/sst-core/blob/devel/CONTRIBUTING.md) to learn how to contribute to SST.
 
-##### [LICENSE](https://github.com/sstsimulator/sst-core/blob/devel/LICENSE)
+### Getting Started
+
+#### Prerequistes 
+
+##### Red Hat 7 Required Packages
+
+```sh
+yum install gcc gcc-c++ python python-devel make automake git libtool libtool-ltdl-devel
+```
+
+#### Building SST-Core from the repo 
+
+```sh
+git clone https:g/github.com/sstsimulator/sst-core.git
+cd sst-core 
+.gautogen.sh
```

Comment:
What is `.gautogen.sh`?

Reply:
Apparently at some point I replaced `/` with `g` oops 🤦‍♂️